display.php

```diff
@@ -1,117 +1,90 @@
- <?php
-/*
- *Author:mfsi_krishnadev
- *importing files: database connection, image upload & insertion operation
- *purpose: listing of data from the database
-*/
+<?php 
 
-ini_set('display_errors', '1');
-
-include('db_conn.php');
-//include('image_upload.php');
-//require('insert.php');
+ini_set("display_error","1");
+session_start();                            
+include('connection.php');
+//include('user.php');
```

Comment:
dead codes present. remove codes which are not required.
check everywhere and the codes which are comment and not in use then remove those.